cmd/traffic/cmd/manager/state/state.go

```diff
@@ -86,14 +89,21 @@ type interceptFinalizerCall struct {
 	ctx   context.Context
 }
 
+type (
```

Comment:
We don't usually group types. I suspect that if you run `make format`, this grouping will be removed.

Reply:
hum, I don't group them. The lint is [failing](https://github.com/telepresenceio/telepresence/actions/runs/6748003557/job/18345398661)